test/helpers/vscodefake/extensioncontext.ts

```diff
@@ -0,0 +1,18 @@
+import * as vscode from 'vscode';
+import {TestMemento} from './memento';
+
+export class FakeContextDefinition implements vscode.ExtensionContext {
+
+  subscriptions: {dispose(): any;}[];
```

Comment:
This field is not initialized

Reply:
Add throwing exception for all unused parts of the interface.